deno.json

```diff
@@ -6,7 +6,52 @@
     "noImplicitOverride": true,
     "noUncheckedIndexedAccess": true
   },
-  "importMap": "./import_map.json",
+  "imports": {
+    "@deno/doc": "jsr:@deno/doc@0.137",
+    "@deno/graph": "jsr:@deno/graph@^0.74",
+    "@std/archive": "jsr:@std/archive@^0.225.0",
+    "@std/assert": "jsr:@std/assert@^1.0.2",
+    "@std/async": "jsr:@std/async@^1.0.3",
+    "@std/bytes": "jsr:@std/bytes@^1.0.2-rc.3",
+    "@std/cli": "jsr:@std/cli@^1.0.3",
+    "@std/collections": "jsr:@std/collections@^1.0.5",
+    "@std/crypto": "jsr:@std/crypto@^1.0.2-rc.1",
+    "@std/csv": "jsr:@std/csv@^1.0.1",
+    "@std/data-structures": "jsr:@std/data-structures@^1.0.1",
+    "@std/datetime": "jsr:@std/datetime@^0.224.5",
+    "@std/dotenv": "jsr:@std/dotenv@^0.225.0",
+    "@std/encoding": "jsr:@std/encoding@^1.0.1",
+    "@std/expect": "jsr:@std/expect@^1.0.0",
+    "@std/fmt": "jsr:@std/fmt@^1.0.0",
+    "@std/front-matter": "jsr:@std/front-matter@^1.0.1",
+    "@std/fs": "jsr:@std/fs@^1.0.1",
+    "@std/html": "jsr:@std/html@^1.0.1",
+    "@std/http": "jsr:@std/http@^1.0.2",
+    "@std/ini": "jsr:@std/ini@^1.0.0-rc.3",
+    "@std/internal": "jsr:@std/internal@^1.0.1",
+    "@std/io": "jsr:@std/io@^0.224.4",
+    "@std/json": "jsr:@std/json@^1.0.0",
+    "@std/jsonc": "jsr:@std/jsonc@^1.0.0",
+    "@std/log": "jsr:@std/log@^0.224.5",
+    "@std/media-types": "jsr:@std/media-types@^1.0.2",
+    "@std/msgpack": "jsr:@std/msgpack@^1.0.0",
+    "@std/net": "jsr:@std/net@^1.0.0",
+    "@std/path": "jsr:@std/path@^1.0.2",
+    "@std/regexp": "jsr:@std/regexp@^1.0.0",
+    "@std/semver": "jsr:@std/semver@^1.0.1",
+    "@std/streams": "jsr:@std/streams@^1.0.1",
+    "@std/testing": "jsr:@std/testing@^1.0.0",
+    "@std/text": "jsr:@std/text@^1.0.2",
+    "@std/toml": "jsr:@std/toml@^1.0.0",
+    "@std/ulid": "jsr:@std/ulid@^1.0.0",
+    "@std/url": "jsr:@std/url@^0.225.0",
+    "@std/uuid": "jsr:@std/uuid@^1.0.0",
+    "@std/webgpu": "jsr:@std/webgpu@^0.224.5",
+    "@std/yaml": "jsr:@std/yaml@^1.0.2",
+    "automation/": "https://raw.githubusercontent.com/denoland/automation/0.10.0/",
+    "graphviz": "npm:node-graphviz@^0.1.1",
+    "npm:/typescript": "npm:typescript@5.4.4"
+  },
```

Comment:
Why did you remove the `importMap` here for `imports`? This will probably need to be reverted.

Reply:
A `"@std/functions": "jsr:@std/functions@^0.1.0"` will also probably need to be added to the `importMap` for the later lint error to fix itself.

Reply:
Please revert these changes as @BlackAsLight points. `import_map.json` is shared between `deno.json` and `browser-compat.tsconfig.json`. That is why we don't write these mappings here.

Reply:
added the 
  "importMap": "./import_map.json",
removed by mistake. 
Is that all?